### PR TITLE
Make MeanAggregator use linearly-decaying payout structure

### PR DESCRIPTION
**Pull request by spooktheducks**

### CODE REVIEW

File: evm/v0.5/contracts/tests/MeanAggregator.sol [hunk 1]
```diff
@@ -9,11 +9,10 @@ contract MeanAggregator {
   // Relies on Coordinator's authorization of the oracles (no need to track
   // oracle authorization in this contract.)
 
-  mapping(bytes32 /* service agreement ID */ => uint256) numOracles;
-  mapping(bytes32 /* request ID */ => mapping(address /* oracle */ => bool))
-    reported;
   mapping(bytes32 /* service agreement ID */ => uint256) payment;
+  mapping(bytes32 /* service agreement ID */ => address[]) oracles;
   mapping(bytes32 /* request ID */ => uint256) numberReported;
+  mapping(bytes32 /* request ID */ => mapping(address => uint256)) reportingOrder;
```
Comment (thodges-gh) [inline evm/v0.5/contracts/tests/MeanAggregator.sol]: It doesn't appear that this mapping is ever cleaned up (same for `numberReported` but that's not part of this PR).

Comment (spooktheducks) [reply evm/v0.5/contracts/tests/MeanAggregator.sol]: Fixed
